src/nanoJS.js

```diff
@@ -46,12 +46,24 @@ var nano = function(s){
   },
   toggleClass: function (v) {
     return this.each(function (i) {
-      i.classList.toggle(v);
+      if (i.classList) {
+        i.classList.toggle(v);
+      } else {
+        if (!i.className || i.className.indexOf(v) === -1) {
```

Comment:
`<p class="foo-bar">...`

`$('p').toggleClass('foo')`

Should split on `/\s+/g` and do a check / filter. then join